src/daemons/MetaDaemon.cpp

```diff
@@ -99,6 +103,13 @@ int main(int argc, char *argv[]) {
     // The meta server has only one space, one part.
     partMan->addPart(0, 0, std::move(peersRet.value()));
 
+    auto clusterMan
+        = std::make_unique<nebula::meta::ClusterManager>(FLAGS_clusterHosts, FLAGS_clusterid_file);
```

Comment:
here should judge the FLAGS_clusterHosts

Reply:
fixed